### PR TITLE
Omnibus collateral 6 

### PR DESCRIPTION
**Pull request by ChandlerDeKock**

### CODE REVIEW

File: Collateral Omnibus 6.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+## Summary (2-5 sentences)
+
+This UMIP proposes adding **miMATIC**, **VOL** for use as collateral in UMA contracts.
+
+## Motivation
+
+The addition of this collateral currency offers additional functionality to the UMA protocol and increases the range of contracts that can be built.
+
+miMATIC also known as MAI is a soft pegged stablecoin that is backed by other assets in the Polygon network.
+
```
Comment (therealhexi) [inline Collateral Omnibus 6.md:18]: What about VOL? What is the project / token?

**Review by therealhexi — APPROVED** [review]
Minor comments on adding more context

File: Collateral Omnibus 6.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+## Technical Specification
+
+To accomplish this upgrade, the following changes needs to be added to the collateral currency whitelist introduced in UMIP-8.
+
+## Ethereum
+### VOL token
+-   The **VOL** Ethereum [token address](https://etherscan.io/token/0x5166e09628b696285e3a151e84fb977736a83575): 0x5166e09628b696285e3a151e84fb977736a83575
+- A final fee of **850 VOL** needs to be added in the Store contract.
```
Comment (Reinis-FRP) [inline Collateral Omnibus 6.md:26]: ~$280 at current prices is bit too low

File: Collateral Omnibus 6.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+-   A final fee of **23 000 ICE** needs to be added in the Store contract.
+
+### IRON stablecoin
+ - The **IRON** Polygon [token address](https://polygonscan.com/token/0xD86b5923F3AD7b585eD81B448170ae026c65ae9a): 0xD86b5923F3AD7b585eD81B448170ae026c65ae9a
+ - A final fee of **400 IRON** needs to be added in the Store contract.
+
+
+## Rationale
+
+This store fee was chosen as it is approximately equivalent to $400 in line with other collateral currencies as determined by assuming that the MIA and IRON tokens will hold its peg to $1 per token or by using CoinMarketCap and CoinGecko prices.
```
Comment (Reinis-FRP) [inline Collateral Omnibus 6.md:48]:
```suggestion
This store fee was chosen as it is approximately equivalent to $400 in line with other collateral currencies as determined by assuming that the MAI and IRON tokens will hold its peg to $1 per token or by using CoinMarketCap and CoinGecko prices. 
```

**Review by Reinis-FRP — DISMISSED** [review]

**Review by Reinis-FRP — APPROVED** [review]

**Pull request merged**